.travis.yml

```diff
@@ -5,10 +5,11 @@ python:
   - "3.8"
 install:
   - pip install --upgrade pip
-  - pip install numpy jaxlib tensorflow tensorflow-datasets --upgrade
+  - pip install pygame==2.0.0.dev6 tfp-nightly tfds-nightly numpy jaxlib jax tf-nightly matplotlib more-itertools --upgrade
```

Comment:
FYI, I'm still quite confused wrt to why we need `pygame`, `matplotlib` and `more-itertools` since none of these appear to be used anywhere